paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -837,30 +838,70 @@ class PaymentSheet internal constructor(
          * The border color of the primary button.
          */
         @ColorInt
-        val border: Int
+        val border: Int,
+        /**
+         * The background color for the primary button when in a success state. If null,
+         * defaults to base green background color.
+         */
+        @ColorInt
+        val successBackgroundColor: Int = PRIMARY_BUTTON_SUCCESS_BACKGROUND_COLOR.toArgb(),
+        /**
+         * The success color for the primary button text when in a success state. If null,
+         * defaults to `onBackground` value.
+         */
+        @ColorInt
+        val onSuccessBackgroundColor: Int? = null,
```

Comment:
Let’s remove the nullability and enforce that it defaults to `onBackground`. This way, we can avoid the nullability handling a few layers down.

```suggestion
        /**
         * The color for the primary button content when in a success state. Defaults to `onBackground`.
         */
        @ColorInt
        val onSuccessBackgroundColor: Int = onBackground,
```